server/etcdserver/bootstrap_test.go

```diff
@@ -245,7 +250,7 @@ func prepareData(cfg config.ServerConfig) (err error) {
 func createWALFileWithSnapshotRecord(cfg config.ServerConfig, snapshotTerm, snapshotIndex uint64) (err error) {
```

Comment:
The `err` is used in defer so keep it here.

Reply:
ack